ntoskrnl/io/pnpmgr/pnpinit.c

```diff
@@ -24,6 +24,18 @@ PUNICODE_STRING PiInitGroupOrderTable;
 USHORT PiInitGroupOrderTableCount;
 INTERFACE_TYPE PnpDefaultInterfaceType;
 
+ARBITER_INSTANCE IopRootBusNumberArbiter;
+ARBITER_INSTANCE IopRootIrqArbiter;
+ARBITER_INSTANCE IopRootDmaArbiter;
+ARBITER_INSTANCE IopRootMemArbiter;
+ARBITER_INSTANCE IopRootPortArbiter;
+
+NTSTATUS NTAPI IopPortInitialize();
+NTSTATUS NTAPI IopMemInitialize();
+NTSTATUS NTAPI IopDmaInitialize();
+NTSTATUS NTAPI IopIrqInitialize();
+NTSTATUS NTAPI IopBusNumberInitialize();
```

Comment:
`()` here means "any number of parameters. Should be `(VOID)`.

Reply:
Ok.